src/utils/fragment-tools.ts

```diff
@@ -65,6 +65,7 @@ export const getAllComponentStyleClasses = (componentObj: any, fragments: any[])
 
 			styleClasses = {
 				...styleClasses,
+				// eslint-disable-next-line @typescript-eslint/no-use-before-define
```

Comment:
This has a smell... Why is this needed?
What other way do we have for solving this?

Reply:
'getAllFragmentStyleClasses' was used before it was defined. It still finds the function, but eslint doesnt like it so i disabled the rule for that line.

Reply:
Right, and if we change the order, the other one is gonna be problematic...

I found a way to make it go away (declare it before, then reassign it, then export), but I'm not sure it's cleaner...

```suggestion
				// we can't avoid this without a messy declare+reassign+export
				// eslint-disable-next-line @typescript-eslint/no-use-before-define
```